### PR TITLE
Web 51

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: docs/1.0.0-beta-1/metrics.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Since multiple processes can report the same metrics to services like Graphite
+or Ganglia, each process adds a unique id.  When running in yarn, this id is of
+the format `worker-<instance id>` or `oracle-<instance id>`.  When not running
+from yarn, this id consist of a hostname and a base36 long thats unique across
+all fluo processes.  In the table below this composite id is represented with
+`<pid>`. 
+
+|Metric                                   | Type           | Description                         |
+|-----------------------------------------|----------------|-------------------------------------|
+|i.f.&lt;pid&gt;.tx.lockWait.&lt;cn&gt;               | [Timer][T]     | *WHEN:* After each transaction. *COND:* &gt; 0 *WHAT:* Time transaction spent waiting on locks held by other transactions.   |
```
Comment (keith-turner) [inline docs/1.0.0-beta-1/metrics.md]: Do you know if jeklly renders this as a table?

Comment (mikewalch) [inline docs/1.0.0-beta-1/metrics.md]: Yes it does.  I did have to add the following to `_config.yml` which is included in this PR:

```
redcarpet:
  extensions: ["tables"]
```

File: _scripts/convert-docs.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      print >> fout, "---"
+
+      if inPath.endswith("README.md"):
+        fin.readline()
+        fin.readline()
+        print >> fout, "\n**Please Note** - This documentation is for the latest Fluo release.  Documentation for prior releases can be found [here](/docs/)"
+
+      for line in fin:
+        if line.startswith("["):
+          if line.find(".md") != -1:
```
Comment (keith-turner) [inline _scripts/convert-docs.py]: Should make the script warn for anchors links like `foo.md#bar` that work on github

File: docs/1.0.0-beta-1/applications.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+`com.SimpleLoader` collided with another transaction on row `r1` and column
+`fam1 qual1`.
+
+```
+2015-02-11 18:17:02,639 [tx.collisions] TRACE: txid: 106 class: com.SimpleLoader
+2015-02-11 18:17:02,639 [tx.collisions] TRACE: txid: 106 collisions: {r1=[fam1 qual1 ]}
+```
+
+[phrasecount]: https://github.com/fluo-io/phrasecount
+[FluoFactory]: https://github.com/fluo-io/fluo/blob/1.0.0-beta-1/modules/api/src/main/java/io/fluo/api/client/FluoFactory.java
```
Comment (keith-turner) [inline docs/1.0.0-beta-1/applications.md]: Could make the script rewrite these links to the javadoc

File: _scripts/convert-docs.py (hunk 2 above)
Comment (mikewalch) [inline _scripts/convert-docs.py]: fixed in b73c7ecffa82b

File: docs/1.0.0-beta-1/applications.md (hunk 3 above)
Comment (mikewalch) [inline docs/1.0.0-beta-1/applications.md]: fixed  in b73c7ecffa82b

**Comment by keith-turner** [comment]
+1

**Pull request merged**